slothy/targets/arm_v7m/arch_v7m.py

```diff
@@ -154,25 +154,27 @@ def start(self, loop_cnt, indentation=0, fixup=0, unroll=1, jump_if_empty=None,
         logging.debug(f"Loop counter {loop_cnt} is incremented by {inc_per_iter} per iteration")
         # Check whether instructions modifying the loop count moved to
         # pre/postamble and adjust the fixup based on that.
-        new_fixup = 0
-        if postamble_code is not None:
-            new_fixup = 0
-            for l in postamble_code:
-                if l.text == "":
-                    continue
-                inst = Instruction.parser(l)
-                if loop_cnt in inst[0].args_in_out and inst[0].increment is not None:
-                    new_fixup = new_fixup + simplify(inst[0].increment)
-
-        if new_fixup != 0 or fixup != 0:
+        # new_fixup = 0
```

Comment:
@dop-amin Why is this merely commented out? Is going to come back? If not, can we just remove it?